_bioinfcomputing/software_overview.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Overview of Software Development at Fred Hutch
+last_modified_at: 2019-04-28
+primary_reviewers: k8hertweck
+---
+
+
+Frequently in scientific research, we must write code in order to perform our desired analysis. Whether we are writing a short script for our own manuscript, or making an app we intend to share with other researchers, incorporating standard programming practices will enhance the readability and reproducibility of our work. Below, we provide an overview of Fred Hutch coding standards as well as resource pages for commonly used programming languages in biosciences research.
+
+## [Getting Started with Common Languages](/bioinfcomputing/software_languages/)
+
+Fred Hutch has several resources available for folks that are seeking instruction in a new programming language, version control (managing and sharing code), or navigating linux/shell scripting. If software development in research is new to you, or you wish to pick up a new skill, we recommend you start with the [Common Languages](/bioinfcomputing/software_languages/) page.
+
+The remainder of this section will presume a basic skill level in at least one common language, as well as a basic understanding of [GitHub](/bioinfcomputing/software_managecode/).
```

Comment:
I think this does a great job accomplishing the goal we discussed previously: managing expectations for both experienced users and newbies.